spec/models/item_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe Item do
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:description) }
+    it { is_expected.to validate_presence_of(:unit_price_in_cents) }
+    it { is_expected.to validate_presence_of(:merchant_id) }
+    it { is_expected.to validate_presence_of(:updated_at) }
+    it { is_expected.to validate_presence_of(:created_at) }
+  end
+
+  describe 'relationships' do
+    it { is_expected.to respond_to(:merchant) }
```

Comment:
What would you think about using `belongs_to` instead of `respond_to`? Same in Merchant spec